docker/README.md

```diff
@@ -26,13 +26,20 @@ Run all docker-compose commands from this directory.
     export FIXUID=$(id -u)
     export FIXGID=$(id -g)
     ```
-2. This recreates most of the SETUP.md instructions, and will probably take a long time:
+2. If you haven't already, add the following to your locals.yml file:
+
+   ```
+   build_apps: true
+   use_my_apps: true
```

Comment:
rather than asking the user to do this, I think we should just change the value in locals.yml.default, which is automatically copied over to locals.yml during setup if it does not exist already.